lib/salus/scan_report.rb

```diff
@@ -130,7 +133,7 @@ def render_banner(use_colors:)
       status = passed? ? 'PASSED' : 'FAILED'
       status = colorize(status, (passed? ? :green : :red)) if use_colors
 
-      banner = "==== #{@scanner_name}: #{status}"
+      banner = "==== #{@scanner_name} #{@version}: #{status}"
```

Comment:
```suggestion
      banner = "==== #{@scanner_name} v#{@version}: #{status}"
```

It might be nice for the stdout text to include a `v` in front of the version number for clarity and to also to match the other stdout lines like `==== Salus Scan v2.10.13 for ...`. 

This is a nitpick though 😄 ⛏️

Reply:
I added the `v`.  Here's how the stdout looks like now
```
==== BundleAudit v0.6.1: PASSED in 2.99s

==== Gosec v2.4.0: PASSED in 1.99s

==== PatternSearch v0.9.0: PASSED in 0.0s

==== RepoNotEmpty: PASSED in 0.0s

==== Semgrep v0.14.0: PASSED in 0.0s

==== YarnAudit v1.22.0: PASSED in 1.08s

==== ReportNodeModules: PASSED in 0.07s

==== ReportRubyGems: PASSED in 0.01s
```